app/Config/bootstrap.php

```diff
@@ -94,8 +94,8 @@
  * ));
  */
 Configure::write('Dispatcher.filters', array(
-	'AssetDispatcher',
-	'CacheDispatcher'
+    'AssetDispatcher',
+    'CacheDispatcher'
```

Comment:
__[staging][code_sniffer]__
* Comma required after last value in array declaration